dynamic_object sDynamicObj
mark_object    sMarkObj
sign_object    sSignObj 